types/artifacts.d.ts

```diff
@@ -393,6 +393,15 @@ declare global {
          */
         fmpFellBack: boolean;
       }
+
+      export type UserTimings = {
```

Comment:
isn't this supposed to be an array of user timings not a keyed object?

Reply:
This way the only way I managed to get type checking done after a while.
If you have an easier solution, please let me know :)